src/main/java/me/jellysquid/mods/sodium/client/model/quad/properties/ModelQuadFacing.java

```diff
@@ -61,7 +61,7 @@ public static ModelQuadFacing fromVector(Vector3f normal) {
                 return WEST;
             }
         }
-        return UNASSIGNED;
+        return UP; // Default in 1.16.5
```

Comment:
This breaks face culling of unaligned quads. This needs to be localized more specifically in `nd.Quad`. On 1.16 it's only `BakedQuad#getFace` that defaults to up.